src/emcpy/stats.py

```diff
@@ -247,3 +247,41 @@ def get_linear_regression(x, y):
     # by y_pred = slope * x + intercept
     y_pred = model.predict(x)
     return y_pred, r_sq, intercept, slope
+
+
+def bootstrap(insample, level=.95, estimator='mean', nrepl=10000):
+    """
+    Generate emprical bootstrap confidence intervals.
+    See https://ocw.mit.edu/courses/mathematics/
+                18-05-introduction-to-probability-and-statistics-spring-2014/
+                readings/MIT18_05S14_Reading24.pdf for more information.
+
+    Args:
+        insample: (array like) is the array from which the estimator (u)
+                  was derived (x_1, x_2,....x_n).
+        level: (float, default=0.95) desired confidence level for CI bounds
+        estimator: (char, default='mean') type of statistic obtained from
+                  the sample (mean or median)
+        nrepl: (integer, default=1000) number of replicates
+
+    Returns:
+        Lower and upper bounds of confidence intervals
+    """
+    if any(_np.isnan(insample)):
+        print('{:s}').format('bootstrap_ci.py: NaN detected. Dropping NaN(s) input prior to bootstrap...')
```

Comment:
Is there a reason why to not just use `print('bootstrap_ci.....')`?

Reply:
No reason in particular.  I made as few changes as I could to the original code.  I'll be glad to simplify this though.

Reply:
Fair enough.  I can clean that up if/when other changes need to be made to that file.